app/views/sponsors/index.html.erb

```diff
@@ -1,31 +1,63 @@
-<h1>All Sponsors</h1>
+<div class="sponsor">
+  <div class="container mt-5 col-pixel-width-900">
+    <div>
+      <h1 class="heading">Virtual Fair</h1>
+    </div>
+    <div class="row d-flex justify-content-between p-4">
```

Comment:
`row` is used to divide into columns which we are not doing here.
and also `d-flex` shouldn't be used with `row` as both already have `flex` display.